artifacts/experimental/etcd-proxy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  name: etcdproxy
+  labels:
+    app: etcdproxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+        app: etcdproxy
+  template:
+    metadata:
+      labels:
+        app: etcdproxy
+    spec:
+      containers:
+      - name: etcdproxy
+        image: quay.io/coreos/etcd:v3.2.18
+        command: ["/usr/local/bin/etcd", "grpc-proxy", "start"]
+        args: ["--endpoints=http://172.17.0.3:2379", "--namespace=/exp", "--listen-addr=0.0.0.0:23790"]
```

Comment:
ip address is not known ahead of time, so this won't work generically.

Reply:
reference the service, not the pod.

Reply:
Resorted to service in #2 https://github.com/xmudrii/etcdproxy-controller/pull/2/files#diff-2f3439ebbf3e4edae0f942275399111cR21